packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -194,10 +203,44 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
   }
 
   protected updateSuggestions(suggestions: any[]) {
-    this.suggestionStore.updateSuggestions(suggestions);
+    this.suggestionStore.updateSuggestions(suggestions, true);
     this.forceUpdate();
   }
 
+  protected onEmptyInputFocus() {
+    let suggestions: T[] | PromiseLike<T[]> = this.props.onInputFocus(this.state.items);
```

Comment:
props.onInputFocus is used so that the user can specify their own method for choosing what to show when focusing an empty people picker.